src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -883,7 +883,7 @@ public function single_row( $item ) {
 			} else {
 				if ( $restrict_network_active ) {
 					$actions = array(
-						'network_active' => __( 'Network Active' ),
+						'network_active' => _x( 'Network Activate', 'plugin' ),
```

Comment:
I believe this one may be incorrect. The `network_active` action should just display "Network Active", whereas it's the `network_activate` action that shows "Network Activate". See [here](https://github.com/costdev/wordpress-develop/blob/7ee50fe6e3ca2950216e8b0b1dbad267e297335c/src/wp-admin/includes/class-plugin-installer-skin.php#L121).

Reply:
Whoops, good catch!